.github/workflows/build-libraries.yaml

```diff
@@ -84,6 +84,9 @@ jobs:
         ATOMVMLIB_FILE=atomvmlib-${{ github.ref_name }}.avm
         mv libs/atomvmlib.avm "libs/${ATOMVMLIB_FILE}" &&
         sha256sum "libs/${ATOMVMLIB_FILE}" > "libs/${ATOMVMLIB_FILE}.sha256"
+        HELLO_WORLD_FILE=hello_world-${{ github.ref_name }}.avm
+        mv examples/erlang/hello_world.avm "examples/erlang/${HELLO_WORLD_FILE}" &&
```

Comment:
I think that GitHub workflows do not need `&&` for joining multiple commands, I think it is implicit.

Reply:
> I think that GitHub workflows do not need `&&` for joining multiple commands, I think it is implicit.

I just copy-pasted the two lines you had previously.  Should that `&&` be removed as well?